js/pulse.js

```diff
@@ -50,10 +57,14 @@
              */
             this.initialize = function(adManagerController, playerId) {
                 amc = adManagerController; // the AMC is how the code interacts with the player
+                //Enable overlay close button right away for Pulse
+                amc.adManagerSettings["showNonLinearCloseButton"] = true;
```

Comment:
@pilievOoyala I should just remove it, right? Has `sendURLToLoadAndPlayNonLinearAd` been updated?